packages/x-components/src/x-modules/history-queries/components/my-history.vue

```diff
@@ -72,8 +91,19 @@
      *
      * @public
      */
-    @Prop()
-    protected animation!: Vue;
+    @Prop({ default: 'ul' })
+    protected animation!: Vue | string;
+
+    /**
+     * The current locale.
+     *
+     * @public
+     */
+    @Prop({
+      default: () =>
+        (this as unknown as Vue & { snippetConfig?: SnippetConfig }).snippetConfig?.lang ?? 'en'
```

Comment:
As I told you privately, this is not working as expected, as `props` are not available at this point in time. We'll have to refactor it using a `computed`.